.github/workflows/go.yml

```
@@ -14,17 +14,26 @@ on:
         go-version: 'stable'
 
+    - name: Run go generate
+      run: go generate ./internal/...
+
     - name: Build
       run: go build -v ./...
 
     - name: Test
       run: go test -v -race ./...
+
+    - namee: Test run
```

actionlint (1.7.4)

unexpected key "namee" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)